classes/aioseop_logging.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * @package All-in-One-SEO-Pack
+ */
+
+/**
+ * Class aioseop_logging
+ *
+ * Class used for logging.
+ */
+// @codingStandardsIgnoreStart
+class aioseop_logging {
+// @codingStandardsIgnoreEnd
+
+	const LOG_LENGTH = 100;
+	const LOG_EXPIRY_MINS = 5;
```

Comment:
there are filters in the constructor

Reply:
So, there's really no use having a constant then? It's inconsistent with the operations if a hook is added to modify the operation. Conceptually, this would just add confusion.